app/views/shared/_accordion.html.slim

```diff
@@ -3,7 +3,7 @@
 )
   .accordion-header(
     class=('display-none' if options[:hide_header])
-    role="heading"
+    role="button"
```

Comment:
So this is a global change that I made...the partners had issues with these according headers being "headers" when they are intended to be clicked on. I thought about making this configurable, but figured since all accordion headers are clickable...they should all be buttons? Open to thoughts here

Reply:
This may be the most efficient solution.

Perhaps you could use an aria-label to read "<header> click this header to toggle this section's content", but I'm not sure this will placate the partners who are against clickable headers being announced as such.

Also, I think aria-label works in different ways depending on the browser/AT.

So again, role=button may have to do for now.